tests/FSharp.Test.Utilities/Compiler.fs

```diff
@@ -488,6 +491,12 @@ module rec Compiler =
     let asLibrary (cUnit: CompilationUnit) : CompilationUnit =
         withOutputType CompileOutput.Library cUnit
 
+    let asNetStandard20 (cUnit: CompilationUnit) : CompilationUnit =
+        match cUnit with
+        | FS fs -> FS { fs with TargetFramework = TargetFramework.NetStandard20 }
+        | CS cs -> CS { cs with TargetFramework = TargetFramework.NetStandard20 }
```

Comment:
Curious - does it work with C# tests too?

Reply:
not yet, when I need it for C# I will make it work.